libcodechecker/log/log_parser.py

```diff
@@ -206,6 +206,21 @@ class ImplicitCompilerInfo(object):
     compiler_target = {}
     compiler_standard = {}
 
+    compiler_isexecutable = {}
+
+    def is_executable_compiler(self, compiler):
```

Comment:
A more general version of this `is_executable` could be made, which could be used at other places too. I think where we detect `scan-build` on osx something similar was done.